purchase_picking_state/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Picking State in Purchase Order
+===============================
+
+This module allows to add picking status on the purchase order
+ 
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+purchase-workflow/issues/new?body=module:%20
+purchase_picking_state%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
@pedrobaeza, Thanks for the review, I correct it.

Le 31/03/2016 09:27, Pedro M. Baeza a écrit :
>
> In purchase_picking_state/README.rst 
> <https://github.com/OCA/purchase-workflow/pull/219#discussion_r58010721>:
>
> > +
> > +.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
> > +   :alt: Try me on Runbot
> > +   :target: https://runbot.odoo-community.org/runbot/142/8.0
> > +
> > +Bug Tracker
> > +===========
> > +
> > +Bugs are tracked on `GitHub Issues
> > +<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
> > +check there if your issue has already been reported. If you spotted it first,
> > +help us smashing it by providing a detailed and welcomed `feedback
> > +<https://github.com/OCA/
> > +purchase-workflow/issues/new?body=module:%20
> > +purchase_picking_state%0Aversion:%20
> > +8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
>
> 9.0
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/OCA/purchase-workflow/pull/219/files/6d00c5054f08ac750e64b55538de563fb50af1cd#r58010721>
>